README.md

```diff
@@ -260,6 +261,9 @@ time.
 
 #### Building with Xcode
 
+Before you start this process, if you have a fresh install of Xcode, open it and sign-in.
```

Comment:
Can we make this a bit more straightforward for people who might not know Xcode. Something like a well-worded "Open Xcode, go to settings, accounts, sign in w/ your Apple ID."